### DISCUSSION TITLE
podman pull ghcr.io/open-webui/open-webui:v0.3.23 fails

### DISCUSSION THREAD

**Discussion by josephtingiris** [open-webui/open-webui#5580] [General]
# Bug Report

## Installation Method

podman pull ghcr.io/open-webui/open-webui:main
-or-
podman pull ghcr.io/open-webui/open-webui:v0.3.23

## Environment

- **Open WebUI Version:** v0.3.23
- **Ollama (if applicable):** n/a

- **Operating System:** n/a
- **Browser (if applicable):** n/a

**Confirmation:**

- [X ] I have read and followed all the instructions provided in the README.md.
- [X] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [X] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

pull succeeds

## Actual Behavior:

pull of main or v0.3.23 fail with

```
Error: copying system image from manifest list: writing blob: storing blob to file "/var/tmp/container_images_storage2080273464/2": pigz: skipping: <stdin>: corrupted -- crc32 mismatch: exit status 1
```

podman pull of dev or v0.3.22 succeed as expected

## Description

**Bug Summary:**

podman pull fails

## Reproduction Details

**Steps to Reproduce:**
[Outline the steps to reproduce the bug. Be as detailed as possible.]

`podman pull ghcr.io/open-webui/open-webui:v0.3.23`

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots/Screen Recordings (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by JeremyGarman** [comment]
Have you tried "podman pull ghcr.io/open-webui/open-webui:**latest**" ?  (rather than  :main  or  :<version>)

**Reply by josephtingiris** [reply]
Yes.  This conversation was created for an issue I opened last September  (#5579).  It's since been fixed and the issue closed.

Both ..

`podman pull ghcr.io/open-webui/open-webui:main`

and

`podman pull ghcr.io/open-webui/open-webui:v0.3.23`

.. are working properly today.
